doc/schema/robot.02.xsd

```diff
@@ -60,13 +60,15 @@
             <xs:element name="doc" type="xs:string" minOccurs="0" />
             <xs:element name="tag" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
             <xs:element name="timeout" type="Timeout" minOccurs="0" />
-            <xs:element name="status" type="Status" />
+            <xs:element name="status" type="SuiteAndTestStatus" />
         </xs:choice>
         <xs:attribute name="name" type="xs:string" />
         <xs:attribute name="id" type="xs:string" />
     </xs:complexType>
     <xs:complexType name="Keyword">
         <xs:choice maxOccurs="unbounded">
+            <!-- These keywords can be regular keywords or test setup and teardown.
+                 If the later, they must have the `type` attribute set accordingly. -->
```

Comment:
What does this comment refer to? Does it refer to the `<xs:complexType name="Keyword">` or keywords it can contain? In the latter case the comment would be pretty strange because keywords inside a keyword cannot be test setups or teardowns. In the former case the comment would be places a bit strangely. Mentioning only test setup/teadown would be also a bit strange when also suites have them and keywords can have teardown. If the comment would be moved a few lines and it would talk only about setups and teardowns (i.e. no word "test") it would make more sense to me.

Reply:
you're right @pekkaklarck  I've updated it